sensors/Makefile

```diff
@@ -4,26 +4,49 @@
 # Copyright 2022 Phoenix Systems
 #
 
+LOCAL_PATH := $(call my-dir)
+
 # Sensors Server API Library
 NAME := libsensors
 include $(static-lib.mk)
 
+# FIXME: should be stored and compiled outside 
```

Comment:
whitespace at the end

not sure it should be stored / compiled outside -  should provide different implementations with common API

Reply:
Now compiled into `libsensors`.

Reply:
IMHO we shouldn't merge target-dependant component into the common code, it should be solved by our buildsystem